docs/faq.md

```diff
@@ -14,24 +14,26 @@ Go to each node and create the ZFS Pool, which will be used for provisioning the
 Once ZFS POOL is created we can install OpenEBS ZFS driver by running the following command.
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/openebs/zfs-localpv/develop/deploy/zfs-operator.yaml
+helm repo add openebs https://openebs.github.io/openebs
+helm repo update
+helm install openebs --namespace openebs openebs/openebs --create-namespace
 ```
 
 Verify that the ZFS driver Components are installed and running using below command :
 
 ```
-$ kubectl get pods -n kube-system -l role=openebs-zfs
+$ kubectl get pods -n openebs -l role=openebs-zfs
```

Comment:
make this change in all other places as well. See HA and custom topology section (5th and 6th)

Reply:
Done.